src/timer/CountDownTimer.jsx

```diff
@@ -9,15 +9,44 @@ import { TimerContext } from './TimerProvider';
  */
 const CountDownTimer = injectIntl((props) => {
   const timer = useContext(TimerContext);
+  const timeString = timer.getTimeString();
   const [isShowTimer, showTimer, hideTimer] = useToggle(true);
   const { intl } = props;
 
+  const generateAccessbilityString = () => {
+    const hours = parseInt(timeString.substring(0, 2), 10);
```

Comment:
We should find a way not to use another "display string" as the source of truth for this. Especially since we're parsing _back_ into an int value. Maybe we can use the timeState object from the TimerProvider or create some other property for this?

Also I'm not seeing how this string would update, it seems to me it would get stuck at whatever timeString is the first time this component renders?